presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -658,6 +658,8 @@ partitions already exist (that use the original column types). The Hive
 connector supports this by allowing the same conversions as Hive:
 
 * ``varchar`` to and from ``tinyint``, ``smallint``, ``integer`` and ``bigint``
+* ``integer`` to ``bigint``, ``real`` and ``double``
+* ``bigint`` to ``real`` and ``double``
```

Comment:
Does the same apply to ORC as well?

Reply:
I can only assure this new behavior for Parquet. Do I remove the changes on the documentation or do I particularize the Parquet changes as an annotation in the "Schema evolution" section?

Reply:
Let's fix the doc to just specify Parquet

Reply:
Doc fixed to specify conversion details with Parquet format